mis_builder/models/mis_report_instance.py

```diff
@@ -544,20 +545,31 @@ def _compute_pivot_date(self):
     )
     hide_analytic_filters = fields.Boolean(default=True)
 
-    @api.onchange("company_id", "multi_company")
+    @api.onchange("multi_company")
```

Comment:
Shall we add a contraint that 
- company_id is not set when multi_company is True and 
- company_id is set when multi_company if False

Reply:
This constraint is not necessary if the company_id field is hidden in single company mode.